### PR TITLE
Use --timestamp rather then --omit-timestamp

### PR DESCRIPTION
**Pull request by rhatdan**
We recieved feedback on the --omit-timestamp that
users would rather specify the timestamp seconds
rather then just use EPOCH.

This PR removes --omit-timestamp from buildah bud
since this has never been released.

We also hide --omit-timestamp from buildah commit
and allow users to continue to use it, but it conflicts
with --timestamp.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### What type of PR is this?

> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation
> /kind failing-test 
> /kind feature
> /kind flake
> /kind other

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Comment by rhatdan** [comment]
@nalind @vrothberg @saschagrunert @mrunalp @mheon @baude @QiWang19 @ashley-cui @giuseppe PTAL

File: imagebuildah/build.go [hunk 1]
```diff
@@ -168,9 +168,9 @@ type BuildOptions struct {
 	SignBy string
 	// Architecture specifies the target architecture of the image to be built.
 	Architecture string
-	// OmitTimestamp forces epoch 0 as created timestamp to allow for
+	// Timestamp forces epoch 0 as created timestamp to allow for
```
Comment (nalind) [inline imagebuildah/build.go:171]: This comment doesn't look right.

**Comment by nalind** [comment]
Minor nit about the documentation of BuildOptions.Timestamp, otherwise LGTM.

**Comment by TomSweeneyRedHat** [comment]
LGTM other than the comment nit that @nalind spotted.

File: imagebuildah/build.go (hunk 1 above)
Comment (rhatdan) [reply imagebuildah/build.go]: Fixed

**Comment by rhatdan** [comment]
@TomSweeneyRedHat Fixed
Once this is merged, can you start cutting v1.16.0

**Comment by TomSweeneyRedHat** [comment]
LGTM
I want to get @saschagrunert #2572 in too before 1.16.  Will look later today if any other vendor dancing is needed.

**Comment by nalind** [comment]
Uh oh, this looks like it's forcing the dates on all content in new layers to be set to the commit-time by default, which is _not_ what we wanted.

**Comment by rhatdan** [comment]
@nalind Can you suggest a fix?

**Comment by nalind** [comment]
@rhatdan #2576 should have fixed that, thanks!

**Pull request merged**